docker-compose.yml

```diff
@@ -15,23 +15,31 @@ services:
     depends_on:
       - postgres
 
+  # Create EE container
+  cuke-master:
+    image: registry2.itci.conjur.net/conjur-appliance-cuke-master:5.0-stable
+    ports:
+      - "443:443"
+    security_opt:
+      - "seccomp:unconfined"
+
   conjur-proxy-nginx:
     build:
       context: .
       dockerfile: Dockerfile.nginx
     entrypoint: nginx-debug -g 'daemon off;'
     environment:
-        TERM: xterm
+      TERM: xterm
     ports:
-    - "8080:80"
-    - "8443:443"
+      - "8080:80"
+      - "8443:443"
```

Comment:
You should try using ports >1024 (8080 and 8443 can be fine here) for local bindings as we talked on Slack (https://www.w3.org/Daemon/User/Installation/PrivilegedPorts.html)

Reply:
👍

Reply:
Is there another port commonly used port I could use (instead of 8443)? B/c running the scripts together, causes a conflict b/c nginx also needs 8443.

Reply:
If not, I could just tear down the containers after every suite of tests run